roles/tm-deepsec-aws-setup/vars/main.yml

```diff
@@ -0,0 +1,34 @@
+# vars file for tm_silent_setup
```

Comment:
nitpicking: These are the the role default parameters, If feel like it would make more sense to call the file `defaults/main.yml`.

Reply:
+1